plexapi/utils.py

```diff
@@ -24,6 +24,11 @@
 except ImportError:
     tqdm = None
 
+try:
+    from functools import cached_property
+except ImportError:
+    from backports.cached_property import cached_property  # noqa: F401
```

Comment:
If you are adding the backports as a dependency then this try/except is redundant as the package does it already

Reply:
but the dependency has a version range: `python_version<="3.7"`. I understand it is not installed on py 3.8

Reply:
```
$ python --version
Python 3.8.16

$ pip install -r requirements.txt
Ignoring backports.cached-property: markers 'python_version <= "3.7"' don't match your environment
Collecting requests
  Using cached requests-2.28.1-py3-none-any.whl (62 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.13-py2.py3-none-any.whl (140 kB)
Collecting charset-normalizer<3,>=2
  Using cached charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2022.12.7-py3-none-any.whl (155 kB)
Installing collected packages: urllib3, idna, charset-normalizer, certifi, requests
Successfully installed certifi-2022.12.7 charset-normalizer-2.1.1 idna-3.4 requests-2.28.1 urllib3-1.26.13

$ pip list
Package            Version
------------------ ---------
certifi            2022.12.7
charset-normalizer 2.1.1
idna               3.4
pip                22.3.1
requests           2.28.1
setuptools         56.0.0
urllib3            1.26.13
```